src/pages/Account/CodeOfConduct.tsx

```diff
@@ -91,73 +127,82 @@ const CodeOfConduct: React.FunctionComponent<PropsType> = ({ location }) => {
     updateTheme(venue);
   }, [venue]);
 
-  if (!venue) {
-    return <>Loading...</>;
+  if (!venueId) {
+    return <>Error: Missing required venueId param</>;
   }
 
-  if (
-    !venue.code_of_conduct_questions ||
-    venue.code_of_conduct_questions.length === 0
-  ) {
-    proceed();
+  if (isLoaded(venue) && !venue) {
+    return <>Error: venue not found for venueId={venueId}</>;
+  }
+
+  if (!venue) {
+    return <LoadingPage />;
   }
 
   const codeOfConductQuestions = IS_BURN
     ? BURN_CODE_OF_CONDUCT_QUESTIONS
     : venue.code_of_conduct_questions;
 
   return (
-    <div className="page-container code-of-conduct-container">
-      <div className="code-of-conduct-form">
+    <div className="CodeOfConduct page-container">
```

Comment:
do we need the page-container?

Reply:
Yeah, most of the styling still comes from it.